velero-plugins/pod/restore.go

```diff
@@ -32,39 +32,44 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	itemMarshal, _ := json.Marshal(input.Item)
 	json.Unmarshal(itemMarshal, &pod)
 	p.Log.Infof("[pod-restore] pod: %s", pod.Name)
+	
```

Comment:
This is in the openshift-migration-plugin version of this:
```
	// ISSUE-61 : removing the node selectors from pods
	// to avoid pod being `unschedulable` on destination
	pod.Spec.NodeSelector = nil
```
@dymurray We need this for migration since a pod with a node selector will fail on restore if the node names are different than at backup time. I imagine we'd still want it for the B/R case, which would mean not conditionalizing this one for "part-of=openshift-migration". I know some customers have wanted to preserve these (in environments where the node names *are* the same), so we may eventually need this to be configurable (perhaps via annotations on the restore resource), but for now we may just want to strip the field in all cases.

Reply:
Resolved this in latest commit